src/vorta/application.py

```diff
@@ -52,7 +52,8 @@ def __init__(self, args_raw, single_app=False):
             self.setStyleSheet(qdarkstyle.load_stylesheet_pyqt5())
 
         args = parse_args()
-        if hasattr(args, 'foreground') and args.foreground:
+        fg_setting = SettingsModel.get(key='foreground').value
+        if hasattr(args, 'foreground') and args.foreground or fg_setting:
```

Comment:
Did you fully test this? There may be an issue with precedence of those logical conditions and it could need some parenthesis.

Reply:
This is outdated now.